var/spack/repos/builtin/packages/py-flameprof/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyFlameprof(PythonPackage):
+    """Flamegraph generator for python’s cProfile stats."""
+
+    homepage = "https://github.com/baverman/flameprof/"
+    pypi     = "flameprof/flameprof-0.4.tar.gz"
+
+    maintainers = ['haampie']
+
+    version('0.4', sha256='dbc86d4190cbbba624f1e0a40f44d9db96138e27534d83c8ef42d420857875a3')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-uwsgi', type=('build', 'run'))
```

Comment:
Where do you see this dependency listed?

Reply:
rqrmnts.txt or something

Reply:
There isn't one, that's why I'm confused.

Reply:
ah okay, https://github.com/baverman/flameprof/blob/master/reqs.txt is what i got

Reply:
but it's not in the tarball indeed...

Reply:
Even if it was in the tarball I would probably ignore it, it isn't used by setup.py.